keyboards/darkproject/kd83a_bfg_edition/src/eeprom_flash.c

```diff
@@ -0,0 +1,693 @@
+/*
```

Comment:
Please delete this file. We won't be accepting this into the repository, as there is a proper EEPROM-on-SPI-Flash algorithm on the `develop` branch.